src/file-selector.ts

```diff
@@ -121,6 +121,10 @@ function flatten<T>(items: any[]): T[] {
 }
 
 function fromDataTransferItem(item: DataTransferItem) {
+    if ('getAsFileSystemHandle' in DataTransferItem.prototype) {
+        return item.getAsFileSystemHandle().then((fileSystemHandle: any) => fileSystemHandle.getFile())
+            .catch((error: { message: any; }) => error.message);
```

Comment:
Any reason you're catching this here?

Reply:
There might be a slight chance that the file disappears during the dragging, which means `getFile()` would fail, for example, if you `rm -rf some-directory/` and the user drags a file in the about-to-be-deleted directory.